logger.js

```diff
@@ -12,10 +12,23 @@ function pinoLogger (opts, stream) {
   }
 
   opts = opts || {}
+
+  opts.customAttributeKeys = opts.customAttributeKeys || {}
+  var attributeKeys = {
+    req: opts.customAttributeKeys.req || 'req',
+    res: opts.customAttributeKeys.res || 'res',
+    err: opts.customAttributeKeys.err || 'err',
+    responseTime: opts.customAttributeKeys.responseTime || 'responseTime'
+  }
```

Comment:
Would you mind to remove the wrapping object?

Reply:
Done, pending commit.